.github/workflows/pack.yml

```diff
@@ -34,10 +34,10 @@ jobs:
         uses: actions/setup-python@v2
         with:
           python-version: 3.8.6
-      - uses: actions/cache@v2
-        with:
-          path: ${{ matrix.projectPath }}/Library
-          key: ${{ matrix.targetPlatform }}-${{ matrix.projectPath }}-Library-v2
+      # - uses: actions/cache@v2
```

Comment:
Are we not using the cache function anymore?

Reply:
Yes we use. This differ does not remained anymore.